coretex/coretex/sample/sequence_sample/local_sequence_sample.py

```diff
@@ -68,3 +114,55 @@ def unzip(self, ignoreCache: bool = False) -> None:
             file_utils.gzipDecompress(compressedSequencePath, decompressedSequencePath)
         except FileNotFoundError:
             pass
+
+    def __unzipPairedEnd(self, ignoreCache: bool = False) -> None:
+        super().unzip(ignoreCache)
+
+        try:
+            compressedForwardPath = getForwardSequenceFile(Path(self.path), [".gz"])
+            compressedReversePath = getReverseSequenceFile(Path(self.path), [".gz"])
+
+            for compressedSequencePath in [compressedForwardPath, compressedReversePath]:
+                decompressedSequencePath = compressedSequencePath.parent / compressedSequencePath.stem
+
+                if decompressedSequencePath.exists() and not ignoreCache:
+                    return
+
+                file_utils.gzipDecompress(compressedSequencePath, decompressedSequencePath)
+        except FileNotFoundError:
+            pass
+
+    def unzip(self, ignoreCache: bool = False) -> None:
+        self.__unzipPairedEnd(ignoreCache) if self.isPairedEnd() else self.__unzipSingleEnd(ignoreCache)
```

Comment:
If a line is longer then we should use normal `if-else` statement instead of single-line one, just for readability